extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheFieldAccessMethodVisitor.java

```diff
@@ -5,6 +5,8 @@
 import org.objectweb.asm.MethodVisitor;
 import org.objectweb.asm.Opcodes;
 
+import io.quarkus.panache.common.deployment.JavaBeanUtil;
```

Comment:
ditto

Reply:
It is used, it's a result of moving this type to a different package, so it did not use to be imported.